README.md

```diff
@@ -3,4 +3,42 @@
 This will be an official client for [Apify API](https://www.apify.com/docs/api/v2).
 It's still work in progress, so please don't use it yet!
 
+TODO: write a proper README
```

Comment:
maybe let's remove this TODO since everybody will see on the first page - not good UX

Reply:
Good point, I'll remove the TODO for now, but we should still write a proper README once the code is nearly done, with some usage examples etc.